exception-handling/raising-and-handling-exceptions.md

```diff
@@ -113,14 +113,14 @@ except ExampleError as error_as_a_variable:
     print(f"An exception occurred. Here are the error details: {error_as_a_variable}")
 ```
 
-| Piece of Code             | Notes                                                                                                                                                                                                                                                      |
+| <div style="width:220px;">Piece of Code</div>            | Notes                                                                                                                                                                                                                                                      |
```

Comment:
It looks great!!!

For what it's worth, as a CSS-opinion-haver...

```suggestion
| <div style="min-width:220px;">Piece of Code</div>            | Notes                                                                                                                                                                                                                                                      |
```

I feel like what's more accurate to our intent is specifying `min-width`. (Also specifying a fixed width with pixels freaks my CSS spidey senses out.)

It produces identical results from what I can see. For your convenience, here's a screenshot of that (from my browser)

<img width="1440" alt="Screen Shot 2021-02-26 at 12 32 15 PM" src="https://user-images.githubusercontent.com/30602862/109351771-ab23a200-782e-11eb-8163-6c63148a91bb.png">

Reply:
Yay! Thank you for figuring this out!